data/json/player_activities.json

```diff
@@ -341,6 +343,7 @@
     "activity_level": "NO_EXERCISE",
     "verb": "driving",
     "suspendable": false,
+    "rooted": true,
```

Comment:
I don't think you should be able to be rooted while driving.

Reply:
I remember thinking for ACT_AUTODRIVE here, "what if there was a planter vehicle part, wouldn't that be cool?"

But you're right, this isn't the PR for that! Will remove shortly.